opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -1160,6 +1160,7 @@ namespace Opm
                 const double thp = WellBhpThpCalculator(*this).calculateThpFromBhp(well_rates_bhp_limit,
                                                                                    bhp_limit,
                                                                                    this->connections_.rho(),
+                                                                                    // TODO: why not [this]() { return well_.wellEcl().alq_value(); } like WellBhpThpCalculator.updateThp()
```

Comment:
Remove this comment?

Reply:
I would like to rephrase the comment, because it is pointing to a problem between the MSW and STDW well implementation. We should have a follow up PR to address that.

Reply:
I moved and rephrased the comment to `WellBhpThpCalculator.cpp`. Since it is not causing problem in this file.